lib/splitclient-rb/clients/split_client.rb

```diff
@@ -208,13 +134,17 @@ def keys_from_key(key)
 
     def parsed_treatment(multiple, treatment_data)
       if multiple
+      {
+        treatment: treatment_data[:treatment],
+        label: treatment_data[:label],
+        change_number: treatment_data[:change_number],
+        configurations: treatment_data[:configurations]
```

Comment:
I tagged you on the GDoc but this one should be "config" instead of configurations (for a single treatment config, what we expose to the customer)